src/core/types/Partner.ts

```diff
@@ -0,0 +1 @@
+export type PartnerId = 'ledger';
```

Comment:
Is the intention here to have multiple partner ids?

Reply:
yes, eventually we will add more partners here.